docs/govuk_2ndline.md

```diff
@@ -93,6 +93,10 @@ Constraints
    4. not be assigned more than `max-on-call-shifts` out-of-hours roles in total
    5. not be on in-hours support in the same week that someone else from their team is also on in-hours support
    6. not be on in-hours support in the week after someone else from their team is also on in-hours support
+   7. not be assigned more than `max_inhours_standby` in-hours roles in total
+   8. not be on in-hours standby support in the same week that someone else from their team is also on in-hours standby support
+   9. not be on out-of-hours support in the same week that someone else from their team is also on out-of-hours support
+   10. not be on out-of-hours support in the week after someone else from their team is also on out-of-hours support
```

Comment:
You've not implemented this constraint - is it needed?  In the new `for` loop for 2.9, it'll be:

```python
                if week != 0:
                    prob += (
                        pulp.lpSum(rota[week, person, role.name] for role in Roles if role.value.oncall) + pulp.lpSum(rota[week - 1, person2, role.name] for role in Roles if role.value.oncall) <= 1
                    )
```

Reply:
Sorry no that's not needed, I've removed it now 👍